_includes/troubleshoot/issues/nopods-in-daemonset.md

```diff
@@ -0,0 +1,62 @@
+## 0/0 Pods in DaemonSet
+
+### Issue:
+
+StorageOS pods not being scheduled and no error is shown in the DS.
+```bash
+{{ page.cmd }} -n storageos get daemonset
+NAME                  DESIRED   CURRENT   READY     UP-TO-DATE   AVAILABLE NODE SELECTOR   AGE
+storageos-daemonset   0         0         0         0            0         <none>          5s
+```
+
+### Reason:
+This indicates that no node in the cluster was selected by the Operator to
+deploy StorageOS.
+
+### Doublecheck
```

Comment:
```suggestion
### Assert
```

@actionbuddha how about Assert instead of Doublecheck?

Reply:
done